daemon/db.c

```diff
@@ -1778,7 +1778,7 @@ bool db_update_their_closing(struct peer *peer)
 		     "UPDATE closing SET their_fee=%"PRIu64", their_sig=x'%s', sigs_in=%u WHERE peer=x'%s';",
 		     peer->closing.their_fee,
 		     tal_hexstr(ctx, peer->closing.their_sig,
-				tal_count(peer->closing.their_sig)),
+				sizeof(*peer->closing.their_sig)),
```

Comment:
How is this possible? It's all fixed size (`secp256k1_ecdsa_signature` + enum) anyway, so this should not change at any point.